applications/accounting/src/main/groovy/org/apache/ofbiz/accounting/accounting/AccountingServices.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * 'License'); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.ofbiz.accounting.accounting
+
+import org.apache.ofbiz.entity.util.EntityUtilProperties
+
+Map getArithmeticSettingsFinAccount() {
+    Map result = success()
+    arithmeticSettings = [:]
+    roundingDecimals = EntityUtilProperties.getPropertyValue('arithmetic', 'finaccount.decimals', '2', delegator)
+    arithmeticSettings.put('roundingDecimals', roundingDecimals)
+    roundingMode = EntityUtilProperties.getPropertyValue('arithmetic', 'finaccount.roundingSimpleMethod', 'roundingMode', 'HalfUp', delegator)
+    arithmeticSettings.put('roundingMode', roundingMode)
+    result.put('arithmeticSettings', arithmeticSettings)
+    return success (arithmeticSettings: arithmeticSettings)
```

Comment:
you can replace this by : 
```
Map arithmeticSettings = [roundingDecimals: EntityUtilProperties.getPropertyValue('arithmetic', 'finaccount.decimals', '2', delegator),
                                               roundingMode: EntityUtilProperties.getPropertyValue('arithmetic', 'finaccount.roundingSimpleMethod', 'roundingMode', 'HalfUp', delegator)]
return success([arithmeticSettings: arithmeticSettings])

Reply:
Thanks Nicolas, for your time and effort.

I will improve this piece of code.